vue/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vue-text-mask",
-  "version": "0.2.0",
+  "version": "0.3.0",
```

Comment:
There's no need to bump the version here. This will have to be major bump, i.e. `vue-text-mask@1.0.0` once we merge it :)